src/Microsoft.ML.Data/DataLoadSave/Binary/Codecs.cs

```diff
@@ -783,10 +764,10 @@ public override void MoveNext()
                     Contracts.Check(++_index < _entries, "reader already read all values");
                 }
 
-                public override void Get(ref DvDateTimeZone value)
+                public override void Get(ref DateTimeOffset value)
                 {
                     Contracts.Assert(!_disposed);
-                    value = new DvDateTimeZone(_ticks[_index], _offsets[_index]);
+                    value = new DateTimeOffset(new DateTime(_ticks[_index]), new TimeSpan(_offsets[_index] * TimeSpan.TicksPerMinute));
```

Comment:
`new TimeSpan(_offsets[_index] * TimeSpan.TicksPerMinute)`  should be `new TimeSpan(0, _offsets[_index], 0)`